gconf/handler.go

```diff
@@ -0,0 +1,142 @@
+package gconf
+
+import (
+	"reflect"
+
+	"github.com/iov-one/weave"
+	"github.com/iov-one/weave/errors"
+	"github.com/iov-one/weave/x"
+)
+
+// OwnedConfig must have an Owner field in protobuf. A configuration update
+// message must be signed by an owner in order to be authorized to apply the
+// change.
+type OwnedConfig interface {
+	Unmarshaler
```

Comment:
I think referring to `Configuration` here is cleaner. It shows we extend the Configuration interface with one more method